stubs/D3DShot/d3dshot/capture_output.pyi

```diff
@@ -0,0 +1,42 @@
+import enum
+from _typeshed import Incomplete
+from collections.abc import Sequence
+from ctypes import _CVoidConstPLike
+from typing_extensions import Literal, TypeAlias
+
+from PIL import Image
+
+_Frame: TypeAlias = Image.Image | Incomplete
+# TODO: Complete types once we can import non-types dependencies
+# See: #5768
+# from torch import Tensor
+# from comtypes import IUnknown
+# import numpy.typing as npt
+# _Frame: TypeAlias = Image.Image | npt.NDArray[np.int32] | npt.NDArray[np.float32] | Tensor
+
+class CaptureOutputs(enum.Enum):
+    PIL: int
+    NUMPY: int
+    NUMPY_FLOAT: int
+    PYTORCH: int
+    PYTORCH_FLOAT: int
+    PYTORCH_GPU: int
+    PYTORCH_FLOAT_GPU: int
+
+class CaptureOutputError(BaseException): ...
+
+class CaptureOutput:
+    backend: CaptureOutput
```

Comment:
Would be great to add a comment here explaining what you stated in https://github.com/python/typeshed/pull/8652#discussion_r963191042

Reply:
Done. Hopefully it's clear enough.

Reply:
LGTM!